tests/test_import_export.py

```diff
@@ -17,6 +17,20 @@
 from tests.data import expected as expected_data, input as input_data
 
 
+def _normalize_xml(dirname):
+    """Removes whitespace from xml files in the given directory."""
+    for dname, _, files in os.walk(dirname):
+        for fname in files:
+            fpath = os.path.join(dname, fname)
+            if not fpath.endswith('.xml'):
+                continue
+            with open(fpath) as f:
+                s = f.read()
+                s = clean_xml(s)
+            with open(fpath, 'w') as f:
+                f.write(str(s))
```

Comment:
That doesn't actually fix it. I moved the cast into clean_xml though.

![screenshot from 2015-10-20 10 37 29](https://cloud.githubusercontent.com/assets/3853/10615839/dd890c40-7716-11e5-8f91-4d2d674f0cdb.png)